node/service/Cargo.toml

```diff
@@ -35,6 +34,7 @@ telemetry = { package = "sc-telemetry", git = "https://github.com/paritytech/sub
 # Substrate Primitives
 sp-authority-discovery = { git = "https://github.com/paritytech/substrate", branch = "master" }
 consensus_common = { package = "sp-consensus", git = "https://github.com/paritytech/substrate", branch = "master" }
+beefy-primitives = { package = "sp-consensus-beefy", git = "https://github.com/paritytech/substrate", branch = "master" }
```

Comment:
```suggestion
sp-consensus-beefy = { package = "sp-consensus-beefy", git = "https://github.com/paritytech/substrate", branch = "master" }
```

Reply:
Yeah. you are absolutely right! And I really would :-) 
But then if I follow my instinct I have to rename all this stuff as well:
- beefy-primitives -> sp-consensus-beefy
- grandpa-primitives -> sp-consensus-grandpa
- beefy -> sc-consensus-beefy
- grandpa -> sc-consensus-grandpa
- babe -> sc-consensus-babe
(for the coherence reason 😃)

So probably I would merge this first, just to let polkadot to compile with current substrate master. 
Then I can do that in a follow up PR